src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -54,6 +54,9 @@ protected function instantiateController($class)
         try {
             return parent::instantiateController($class);
         } catch (\Error $e) {
+            if (0 === strpos($class, '\\')) {
+                @trigger_error(sprintf('The controller definition "%s" must not begin with "\"', $class), E_USER_DEPRECATED);
```

Comment:
Maybe: must not start with a slash

Instead of 

“\” ?

Reply:
TBH it's backslash actually ;)

Reply:
Agree with @OskarStark and you. What about
```suggestion
                @trigger_error(sprintf('The controller definition "%s" must not start with a backslash.', $class), E_USER_DEPRECATED);
```
It looks like `start with smth` is the correct wording: https://www.thefreedictionary.com/words-that-start-with-x .
We should also add a dot, because it's a sentence.